src/pages/Main/MainPagination/PhotoCard.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import Card from 'components/Card/Card';
+
+import { AiOutlineHeart } from 'react-icons/ai';
+import { BsBookmark, BsChat } from 'react-icons/bs';
+
+const API = '/data/Mains/Mains.json';
+
+function PhotoCard() {
+  const [isPhotoCard, setIsPhotoCard] = useState([]);
```

Comment:
```suggestion
  const [photoCards, photoCards] = useState([]);
```
`is-`는 불리언 타입 데이터를 사용할 때 적합한 이름 같아요